library/sketches-library/src/main/java/uk/gov/gchq/gaffer/sketches/datasketches/cardinality/serialisation/json/HllSketchWithValues.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.sketches.datasketches.cardinality.serialisation.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import org.apache.datasketches.hll.HllSketch;
+
+import uk.gov.gchq.gaffer.sketches.datasketches.cardinality.function.IterableToHllSketch;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static uk.gov.gchq.gaffer.sketches.datasketches.cardinality.serialisation.json.HllSketchJsonConstants.DEFAULT_LOG_K;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class HllSketchWithValues {
```

Comment:
Could you add Javadoc to explain to role/use for this new class.

Reply:
Is that helpful?